Sources/AlgoliaSearchClient/Models/Search/Response/SearchResponse/Auxiliary/RenderingContent/FacetOrderContainer.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  FacetOrderContainer.swift
+//  
+//
+//  Created by Vladislav Fitc on 17/03/2021.
+//
+
+import Foundation
+
+public struct FacetOrderContainer: Codable {
+
+  public let facetOrdering: FacetOrdering
+
+  public init(facetOrdering: FacetOrdering = .init()) {
+    self.facetOrdering = facetOrdering
+  }
+
+}
+
+public struct FacetOrdering {
+
+  /// The ordering of facets.
+  public let facets: FacetsOrder
+
+  /// The ordering of facet values, within an individual list.
+  public let values: [Attribute: FacetValuesOrder]
```

Comment:
is this an object or an array?

Reply:
It's a map `Attribute -> FacetValuesOrder`